packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -394,6 +443,77 @@ contract Reserve is
     return _transferGold(to, value);
   }
 
+  /**
+   * @notice Transfer ERC20 token to a whitelisted address subject to reserve spending limits
+   * if the limit is set, othersise the limit is 100%.
+   * @param erc20TokenAddress The token address you're transferring.
+   * @param to The address that will receive the ERC20 token.
+   * @param value The amount of ERC20 token to transfer.
+   * @return Returns true if the transaction succeeds.
+   */
+  function transferErc20Token(address erc20TokenAddress, address payable to, uint256 value)
+    external
+    returns (bool)
+  {
+    require(
+      erc20TokenAddress != registry.getAddressForOrDie(GOLD_TOKEN_REGISTRY_ID),
+      "token address should not equal to celo"
+    );
+    require(isSpender[msg.sender], "sender not allowed to transfer Reserve funds");
+    require(isOtherReserveAddress[to], "can only transfer to other reserve address");
```

Comment:
This constraint is not needed. This function will be called by the broker when it needs to transfer a CA for a buy/sell. The to address will either be the reserve or the Mento trader when selling a stable asset.